scripts/common/preflights.sh

```diff
@@ -657,3 +658,21 @@ function bail_if_unsupported_openebs_to_rook_version() {
     fi
 }
 
+# not allow run the installer/upgrade when kurl version is lower than the previous applied before
+function bail_if_kurl_version_is_lower_previous_config() {
+    if commandExists kubectl; then
+       local current_kurl_version
+       current_kurl_version=$(kubectl -n kurl get configmap kurl-current-config -o jsonpath='{ .data.kurl-version }')
```

Comment:
There is a helper function that gets you the "current" kurl version: https://github.com/replicatedhq/kURL/blob/a341c55ae58d340007ad818123d7f65db2316dc1/scripts/common/kurl.sh#L25-L27

Reply:
oh thank you! our goal should be to add as little new code as possible and use utility functions as much as possible as bash is hard...